x11/grace/files/patch-implicit-Xbae.diff

```diff
@@ -0,0 +1,12 @@
+diff -r -u Xbae_orig/Xbae/Draw.c Xbae/Xbae/Draw.c
+--- Xbae_orig/Xbae/Draw.c	1999-09-10 21:25:37
++++ Xbae/Xbae/Draw.c	2024-06-20 17:07:34
+@@ -236,7 +236,7 @@
+ static void
+ xbaeDrawCellString(mw, row, column, x, y, string, bg, fg)
+ XbaeMatrixWidget mw;
+-int row, column;
++int row, column, x, y;
+ String string;
+ Pixel bg, fg;
```

Comment:
FYI, K&R style declarations like this are going to be removed from the next version of the C standard.

Reply:
Thanks.  That'll may a _much_ more major update, because I think those are everywhere.  We'll deal when that happens, but if so, I may campaign much harder for changing the compiler flags for that one.